content_sync/pipelines/definitions/concourse/theme_assets_pipeline.py

```diff
@@ -61,7 +61,9 @@ class ThemeAssetsPipelineDefinition(Pipeline):
     ).root
     _clear_live_cdn_cache_identifier = Identifier("clear-live-cdn-cache-task").root
 
-    _open_discussions_resource = OpenDiscussionsResource()
+    _open_catalog_resources = [
+        OpenCatalogResource(catalog_url) for catalog_url in settings.OPEN_CATALOG_URLS
+    ]
```

Comment:
@gumaerc can this variable be removed? It doesn't seem to be used anywhere in the pipeline.

Reply:
Removed